_src/getting-started/experienced-developers.md

```diff
@@ -0,0 +1,123 @@
+{
+    "template": "../inc/page.template.source",
+    "subtitle": "Getting Started Guide for Experienced Developers",
+    "toc": true
+}
+---
+
+# Introduction
+
+## Understanding Go kit key concepts
+
+### Go kit is not an MVC framework
+
+If you're coming from Symfony (PHP), Rails (Ruby), Django (Python),
+or one of the many popular MVC-style frameworks out there, the first
+thing you should know is that Go kit is not an MVC framework.
+
+Three primary concepts you'll learn about are:
+1. Transports
+2. Endpoints
+3. Services
+
+With a HTTP microservice, requests enter at no. 1 and flow down to no. 3.
+
+This is a bit of an adjustment, but once you grok the concepts,
+you'll hopefully see that the Go kit design is well suited for modern,
+enterprise-grade microservices.
+
+### Transports
+
+The Transport domain is bound to concrete transports like HTTP or gRPC.
+
+In a world where microservices may support one or more transports,
+this is very powerful; you can support a legacy HTTP API and a newer RPC service,
+all in a single microservice.
+
+When implementing a HTTP API, your routing is defined within a HTTP Transport.
+It is most common to see routes defined in a HTTP Router function like this:
+
+```
+r.Methods("POST").Path("/profiles/").Handler(httptransport.NewServer(
+		e.PostProfileEndpoint,
+		decodePostProfileRequest,
+		encodeResponse,
+		options...,
+))
+```
+
+### Endpoints
+
+An Endpoint is like an action/handler on a Controller; it's where safety and antifragile logic lives.
+
+If you implement two Transports (HTTP and RPC), you might have two methods of sending requests to the same Endpoint.
+
+### Services
+
+Services are where all of the business logic is implemented.
+
+An Endpoint might glue multiple Services together.
+
+### Middlewares
+
+Go kit tries to enforce a strict separation of concerns through use of the middleware (or decorator) pattern.
+
+Middlewares can wrap Endpoints or Services to add functionality such as:
+
+  * Logging
+  * Rate limiting
+  * Load balancing
+  * Distributed tracing
+
+You can even chain multiple Middlewares to an Endpoint or Service.
```

Comment:
A bit about the two type of middlewares (Service and Endpoint) would be useful, and then it would follow to mention how to layer in behavior at the Transport level with Before, After, and Finalizer funcs.